### ISSUE TITLE
offlinedatasci install minicran not working

### ISSUE THREAD

**Issue by jsteyn** [carpentriesoffline/offlinedatasci#121]
Trying to install just minicran causes error  saying there is no package called miniCRAN
- using R 4.3.2
- using python 3.10.4

**Comment by hturner** [comment]
This is on Windows 10:

```
offlinedatasci install minicran /install/path                              
Loading required package: miniCRAN                                                                                      
Warning message:                                                                                                        
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :                                   
  there is no package called 'miniCRAN'                                                                                 
Loading required package: remotes                                                                                       
Error: Failed to install 'miniCRAN' from GitHub:                                                                          
  Timeout was reached: [api.github.com] SSL/TLS connection timeout                                                      
Execution halted
```

**Comment by ethanwhite** [comment]
OK, thanks. I think is all stemming from the fact that miniCRAN went through a long period of zero maintenance (requiring us to fork it) combined with some upstream changes. Fortunately minicran has finally been updated so let's switch back to installing from CRAN and see if that helps.

**Comment by abhidg** [comment]
Getting this here, will try setting a mirror and see if it works after that

```
❯ offlinedatasci install minicran tmp
Loading required package: miniCRAN
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called ‘miniCRAN’
Loading required package: remotes
Error in contrib.url(repos, "source") :
  trying to use CRAN without setting a mirror
Calls: install.packages -> contrib.url
In addition: Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called ‘remotes’
Execution halted
```

**Comment by ethanwhite** [comment]
I just merged #122 which should fix this and also adds automated testing to catch this earlier next time. Re-install from the main branch (docs available in the README if you need them) and let me know if that fixes things.

**Comment by ethanwhite** [comment]
If someone can confirm that this is fixed I'll roll a new release

**Comment by abhidg** [comment]
Getting a slightly different error:

```
❯ offlinedatasci install r-packages tmp
Loading required package: miniCRAN
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called ‘miniCRAN’
Error in contrib.url(repos, "source") :
  trying to use CRAN without setting a mirror
Calls: install.packages -> contrib.url
Execution halted
```

**Comment by ethanwhite** [comment]
Thanks @abhidg. I assume the test is failing for you locally as well?

**Comment by ethanwhite** [comment]
#127 will hopefully fix this new error. Still trying to figure out how to add a test for it, so would appreciate someone this is failing for checking if the fix works.

**Comment by ethanwhite** [comment]
There are further improvements to this fix in #129 - so that's the one to test if folks get the chance.

**Comment by hturner** [comment]
I've tested `install r-packages` with PR 129 and that fixes this issue. However, it's not completely working so I'll raise a new issue about that.

**Comment by ethanwhite** [comment]
Thanks @heather! (and thanks for the catch on that other issue as well; more over there soon)

**Comment by ethanwhite** [comment]
Fixed by #122 and #127
